running.md

```diff
@@ -160,18 +160,24 @@ The standard Fedora CoreOS image does not contain Cockpit packages.
 
 2. Reboot
 
-3. Run the Cockpit web service with this privileged container (as root):
+3. Enable password based SSH logins, unless you only use [SSO logins](https://cockpit-project.org/guide/latest/sso.html):
+   ```
+   echo 'PasswordAuthentication yes' | sudo tee /etc/ssh/sshd_config.d/02-enable-passwords.conf
+   sudo systemctl try-restart sshd
+   ```
+
+4. Run the Cockpit web service with this privileged container (as root):
    ```
    podman container runlabel --name cockpit-ws RUN docker.io/cockpit/ws
    ```
 
-4. Make Cockpit start on boot:
+5. Make Cockpit start on boot:
    ```
    podman container runlabel INSTALL docker.io/cockpit/ws
    systemctl enable cockpit.service
    ```
 
-_Steps 3 and 4 are optional if the CoreOS machine will only be connected to from another host running Cockpit._
+_Steps 3 to 5 are optional if the CoreOS machine will only be connected to from another host running Cockpit._
```

Comment:
When looking at the preview this morning, I realized it's kind of odd to have "optional" _after_ the steps. (It's a problem now, but it's even more noticeable with more points.)

Solutions (at least one or both):
- Move the optional message above the optional steps.
- State the step is optional in the step.

Reply:
Note that this was just a straight copy from https://github.com/cockpit-project/cockpit/blob/master/containers/ws/README.md . I updated the text as per your suggestions now. Once we land this, we need to sync this back to cockpit's container README, cockpit-container.git, and the docker.io/quay.io container descriptions.